version

```diff
@@ -0,0 +1 @@
+1.40.0.0
```

Comment:
Discussed offline, if we make this a json file, we have options in the future to make the file more extensible, without breaking existing consumers of the file.